src/update.c

```diff
@@ -190,7 +190,7 @@ static int update_loop(struct list *updates, struct manifest *server_manifest)
 	return ret;
 }
 
-int add_included_manifests(struct manifest *mom, struct list **subs)
```

Comment:
Did this get split after making the changes? Do you recommend me reviewing just the final diff?